research/06-require-ana.tex

```diff
@@ -51,13 +51,13 @@ \subsubsection{Must have}
 also be considered an acronym for the Minimum Usable SubseT.
 
 \paragraph{Interactive shell} Per the definition of a REPL given in
-\cref{sec:repl}, the generated REPLs have to be interactive. It should evaluate
-single statements and expressions typed in by the user and print the results
-back to them.
+\cref{sec:repl}, the REPL has to be interactive. It should evaluate single
+statements and expressions typed in by the user and print the results back to
+them.
 
-\paragraph{Generated from language definition} Every service generated by
-Spoofax is made from the language definitions. It is evident that the REPL
-should be generated from the same definitions if it is to fit within Spoofax.
+\paragraph{Operates with language definition} Every service in Spoofax operates
```

Comment:
Since the title changes, I think it should now read `Operates with language definitions` (plural).